Makefile

```diff
@@ -23,7 +23,7 @@ clean:
 	rm -rf vendor
 
 test: deps
-	@go test $(BUILDFLAGS) -cover $(PACKAGES)
+	@go test $(BUILDFLAGS) -cover $(PACKAGES) ./...
```

Comment:
> ok so the issue was that I was running it from skopeo/vendor for quickly verifying a patch.

How? `github.com/projectatomic/skopeo/vendor/github.com/containers/image` does not contain a copy of this `Makefile` at all.